### DISCUSSION TITLE
Failed to load community plugin in obsidian

### DISCUSSION THREAD

**Discussion by pryndor** [community/community#40675] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

                
2507998 bytes downloaded from github.com at 3.78 Mbps
2507998 bytes downloaded from cloud.githubusercontent.com at 3.58 Mbps
2507998 bytes downloaded from avatars.githubusercontent.com at 44.29 Mbps
2507998 bytes downloaded from github.githubassets.com at 4.52 Mbps
2507998 bytes downloaded from australiaeast.github-debug.com at 7.56 Mbps
2507998 bytes downloaded from brazilsouth.github-debug.com at 1.64 Mbps
2507998 bytes downloaded from centralindia.github-debug.com at 7.46 Mbps
2507998 bytes downloaded from fra.github-debug.com at 4.44 Mbps
2507998 bytes downloaded from iad.github-debug.com at 3.86 Mbps
2507998 bytes downloaded from japaneast.github-debug.com at 7.90 Mbps
2507998 bytes downloaded from koreacentral.github-debug.com at 9.87 Mbps
2507998 bytes downloaded from sea.github-debug.com at 5.76 Mbps
2507998 bytes downloaded from southeastasia.github-debug.com at 10.09 Mbps
2507998 bytes downloaded from github-cloud.s3.amazonaws.com at 4.33 Mbps

                
157.49.254.234

                
Mozilla/5.0 (X11; Linux x86_64; rv:107.0) Gecko/20100101 Firefox/107.0

                
4906006a259b7e55b604ee5b15ad4ac4

                
EBEA:24C7:0AC0:938B1:638AF13A

                
                

                  
traceroute to 157.49.254.234 (157.49.254.234), 30 hops max, 60 byte packets
 1  *
 2  *
 3  *
 4  *
 5  *
 6  *
 7  *
 8  *
 9  *
10  *
11  *
12  *
13  *
14  *
15  *
16  *
17  *
18  *
19  *
20  *
21  *
22  *
23  *
24  *
25  *
26  *
27  *
28  *
29  *
30  *

                  
PING 157.49.254.234 (157.49.254.234) 56(84) bytes of data.

--- 157.49.254.234 ping statistics ---
3 packets transmitted, 0 received, 100% packet loss, time 2055ms

                  
Start: 2022-12-02T22:53:07-0800
HOST: e5973ff74adda0cdd444c88380120546               Loss%   Snt   Last   Avg  Best  Wrst StDev

                  
Host 234.254.49.157.in-addr.arpa. not found: 3(NXDOMAIN)

**Comment by ossanna16** [comment]
Hi there @Bala-periannan and welcome to our community! Can you please elaborate further on your issue/question?

**Comment by pryndor** [comment, marked as answer]
The issue is resolved. Thanks for asking.

Regards,
Bala.
